Ginger/GingerCore/ALM/RQM/ImportFromRQM.cs

```diff
@@ -924,43 +925,44 @@ private static void SaveItemFields(ObservableList<ExternalItemFieldBase> refresh
 
         public static ObservableList<ExternalItemFieldBase> GetOnlineItemFields(BackgroundWorker bw)
         {
+
             ObservableList<ExternalItemFieldBase> fields = new ObservableList<ExternalItemFieldBase>();
 
-            //TODO : receive as parameters:
 
-            RqmRepository rqmRep = new RqmRepository(RQMCore.ConfigPackageFolderPath);
-            List<IProjectDefinitions> rqmProjectsDataList;
-            string rqmSserverUrl = ALMCore.DefaultAlmConfig.ALMServerURL + "/";
-            LoginDTO loginData = new LoginDTO() { User = ALMCore.DefaultAlmConfig.ALMUserName, Password = ALMCore.DefaultAlmConfig.ALMPassword, Server = ALMCore.DefaultAlmConfig.ALMServerURL };
+            try
+            {
+                //TODO : receive as parameters:
 
-            //------------------------------- Improved solution
+                RqmRepository rqmRep = new RqmRepository(RQMCore.ConfigPackageFolderPath);
+                List<IProjectDefinitions> rqmProjectsDataList;
+                string rqmSserverUrl = ALMCore.DefaultAlmConfig.ALMServerURL + "/";
+                LoginDTO loginData = new LoginDTO() { User = ALMCore.DefaultAlmConfig.ALMUserName, Password = ALMCore.DefaultAlmConfig.ALMPassword, Server = ALMCore.DefaultAlmConfig.ALMServerURL };
 
-            string baseUri_ = string.Empty;
-            string selfLink_ = string.Empty;
-            int maxPageNumber_ = 0;
-            int totalCategoryTypeCount = 0;
+                //------------------------------- Improved solution
 
+                string baseUri_ = string.Empty;
+                string selfLink_ = string.Empty;
+                int maxPageNumber_ = 0;
+                int totalCategoryTypeCount = 0;
+
+
+                string categoryValue = string.Empty;  // --> itemfield.PossibleValues.Add(ccNode.Name);
+                string categoryTypeID = string.Empty; //--> itemfield.ID
 
-            string categoryValue = string.Empty;  // --> itemfield.PossibleValues.Add(ccNode.Name);
-            string categoryTypeID = string.Empty; //--> itemfield.ID
-            try
-            {
                 //TODO: Populate list fields with CategoryTypes
+                Reporter.ToLog(eLogLevel.DEBUG, $"Starting fields retrieve process... ");
                 populatedValue = "Starting fields retrieve process... ";
-                if(bw != null)
+                if (bw != null)
```

Comment:
use the function created for populating this message on UI and use it everywhere

Reply:
done